### PR TITLE
adds middleware to convert reponses to camelCase

### PR DESCRIPTION
**Pull request by stkane**
Middleware now converts responses from snake_case to camelCase.

### CODE REVIEW

File: server/arc-client.js [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
 dotenv.config();
 
 const arcadiaApi = axios.create({
   baseURL: "https://api.arcadia.com",
 });
 
+// Axios middleware to convert all api responses to camelCase
+arcadiaApi.interceptors.response.use((response) => {
+  if (
+    response.data
```
Comment (stkane) [inline server/arc-client.js:14]: We can add additional validations here if we need them, but I don't actually think we do at this point

File: server/index.js [hunk 2]
```diff
@@ -83,7 +83,7 @@ app.post("/calculate_counterfactual_bill", async (req, res) => {
     // });
     res.status(200);
   } catch (error) {
-    console.log("oh no we encountered an error!", error.response.data); // TODO: parse HTTP errors if they exists error.response.data.error
+    console.log("oh no we encountered an error!", error); // TODO: parse HTTP errors if they exists error.response.data.error
```
[the remark this answers is not in the document]
Comment (xelzi) [reply server/index.js:86]: hehe yeah this was just debugging!

File: server/arc-client.js [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
 const arcadiaApi = axios.create({
   baseURL: "https://api.arcadia.com",
 });
 
+// Axios middleware to convert all api responses to camelCase
+arcadiaApi.interceptors.response.use((response) => {
+  if (response.data) response.data = humps.camelizeKeys(response.data);
+  return response;
+});
+
```
Comment (xelzi) [inline server/arc-client.js:16]: wonder if this works for nested keys -- not a concern ATM though :)

**Review by xelzi — APPROVED** [review]

File: server/arc-client.js (hunk 3 above)
Comment (stkane) [reply server/arc-client.js:16]: oh yeahhh, it should! At least that's what it says on the package :)

**Pull request merged**